packages/storylite/src/types/story.ts

```diff
@@ -1,17 +1,17 @@
-import { SLComponentProps, SLFunctionComponent, SLNode } from './components'
-import { SLParametersConfig } from './parameters'
+import type { SLComponentProps, SLFunctionComponent, SLNode } from './components'
+import type { SLParametersConfig } from './parameters'
 
-export type SLStoryContext<P extends SLFunctionComponent = SLFunctionComponent<{}>> = {
+export type SLStoryContext<F extends SLFunctionComponent = SLFunctionComponent<{}>> = {
   /**
    * The basic component data.
    *
    * > 💅 This is a StoryLite-only feature.
    */
-  story: BaseStory<P> & { id: string; component: P }
-  args: Story<P>['args']
-  // argTypes: Story<P>['argTypes']
-  // globals: Story<P>['globals']
-  parameters: Story<P>['parameters']
+  story: BaseStory<F> & { id: string; component: F }
+  args: Story<F>['args']
```

Comment:
If a component has no props at all e.g.:

```ts
export default function Form() {}
```

It throws two type errors:

```sh
src/form/form.stories.tsx:16:12 - error TS2322: Type 'unknown' is not assignable to type 'IntrinsicAttributes'.

16           <Story {...context?.args} />
src/form/form.stories.tsx:16:22 - error TS2698: Spread types may only be created from object types.

16           <Story {...context?.args} />
```

This workaround fixes the issue currently:

```ts
export default function Form(_props: FormProps) {}
```